depends_on_stage2

```diff
@@ -143,6 +141,9 @@ def main(check_mode):
     real_extra_dirs = []
     for extra_dir in data["extra_dirs"]:
         real_extra_dir = os.path.realpath(extra_dir)
+        if not check_mode:
+            unshallow(real_extra_dir, data["main_branch"])
```

Comment:
I'm not sure that's correct here as `data` is coming from the main change and not from the `extra_dir`.